IDEAS/Buildings/Components/Shading/Interfaces/PartialShading.mo

```diff
@@ -3,11 +3,38 @@ partial model PartialShading "Window shading partial"
   parameter Boolean controlled=true
     "if true, shading has a control input"
     annotation(Evaluate=true);
+  parameter Modelica.Units.SI.Area A_glazing "Surface area of the glazing";
+  parameter Modelica.Units.SI.Area A_frame "Surface area of the frame";
+  parameter Modelica.Units.SI.Angle inc "Surface inclination";
+  parameter Modelica.Units.SI.Temperature Tenv_nom=280
+    "Nominal temperature of environment";
+  parameter Modelica.Units.SI.Emissivity epsSw_frame
+    "Short wave solar absorption coefficient of the frame";
+  parameter Modelica.Units.SI.Emissivity epsLw_frame
+    "Long wave solar absorption coefficient of the frame";
+  parameter Modelica.Units.SI.Emissivity epsLw_glazing
+    "Long wave solar absorption coefficient of the glazing";
+  parameter Modelica.Units.SI.Emissivity epsSw_shading = 0.5
```

Comment:
Why is this the only emissivity parameter for which a default value is provided?

Reply:
I probably had to do it while testing and then forgot to remove it, thanks!